src/core/requests.ts

```diff
@@ -39,13 +39,14 @@ namespace Requests {
     return <Request>Requests.chain(config.search.defaults, request, config.search.overrides);
   };
 
-  export const pastPurchaseProducts = (state: Store.State, getNavigations: boolean = false): Request => {
+  // tslint:disable-next-line max-line-length
+  export const pastPurchaseProducts = (state: Store.State, getNavigations: boolean = false, options: { pageSize: number, skip: number} = { pageSize: Selectors.pastPurchasePageSize(state), skip: Selectors.pastPurchasePageSize(state) * (Selectors.pastPurchasePage(state) - 1) }): Request => {
```

Comment:
Consider to pass in this long list of parameters as an object or get default value in the body of the function. It causes difficulty to read when it gets long.

Reply:
One question: when you pass in options, what if `pageSize` is defined and `skip` is undefined, it won't get the default value I assume? Or maybe this scenario won't occur?

Reply:
You're right, it won't work. But! If you use this syntax, it does:
`{ pageSize = Selectors.pastPurchasePageSize(state), skip = Selectors.pastPurchasePageSize(state) * (Selectors.pastPurchasePage(state) - 1) }: { pageSize?: number, skip?: number}`

I switched it to not passing it in as default though, because you're right that line's getting too long. :) Just thought that syntax was kewl.